cruise.umple/src/Umple_Code.ump

```diff
@@ -1481,20 +1482,26 @@ class UmpleClass
         regexOperation = regexOperation.replace("_~", "");
         boolean isCurrentMatch = formattedMethod.matches(regexOperation);
 
-        // Determine whether the paremeters of the injection match the parameters
-        // of this method
-/*
         boolean isParameterMatch = true;
-        int i = 0;
-        for(String parameterType : code.getParameters().split(",")) {
-          if(!parameterType.equals(parameters.get(i).getType())) {
-            isParameterMatch = false;
-          }
-          i++;
+
+        if("...".equals(code.getParameters()[opInd])) {
```

Comment:
`code.getParameters()[opInd]` should really be stored in a variable. It's accessed many times.